android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt

```diff
@@ -555,7 +608,7 @@ constructor(
   suspend fun create(vararg resources: Resource) {
     return withContext(dispatcherProvider.io()) {
       resources.onEach { it.generateMissingId() }
```

Comment:
do we need to add `updateLastUpdated` here?

Reply:
that is updated by [the calling method here ](https://github.com/opensrp/fhircore/blob/d505c7e0d23ce287269b7351ea7e7f7c5fa057b5/android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt#L555)